src/marketplace/localserver.ts

```diff
@@ -89,20 +89,22 @@ export class LocalServer {
 
             items.forEach((item) => {
                 if (item.id !== "microsoft.stream-analytics-on-iot-edge") {
-                    const metaData = item.plans[0].artifacts.find((artifact) => artifact.name === "iot-edge-metadata.json");
-                    if (metaData !== null) {
-                        item.iotEdgeMetadataUrl = metaData.uri;
-                    } else {
-                        item.iotEdgeMetadataUrl = null;
-                    }
+                    if (item.plans && item.plans[0] && item.plans[0].artifacts) {
+                        const metaData = item.plans[0].artifacts.find((artifact) => artifact.name === "iot-edge-metadata.json");
+                        if (metaData !== null) {
+                            item.iotEdgeMetadataUrl = metaData.uri;
+                        } else {
+                            item.iotEdgeMetadataUrl = null;
```

Comment:
what will happen if matadata url is null? although not related to the hot fix. This maybe an issue we need take care.